samples/DesktopTests.robot

```diff
@@ -122,3 +122,4 @@ Switch To Desktop Test
     Close Application
     Switch Application      Desktop
     Click Element           name=Start
+    Click Element           name=Start
```

Comment:
Added a second click to avoid leaving the start menu open. This prevents the user from sitting on a screen that does not look like its finished (as the start menu may cover the command prompt running the test.